supabase/__init__.py

```diff
@@ -0,0 +1,6 @@
+__version__ = "0.0.2"
```

Comment:
Shouldn't this be 0.0.3??

Reply:
Yup, good catch. It should be 0.0.3 Thanks!